src/helper/CMakeLists.txt

```diff
@@ -34,7 +37,8 @@ target_link_libraries(sddm-helper
                       Qt${QT_MAJOR_VERSION}::Network
                       Qt${QT_MAJOR_VERSION}::DBus
                       Qt${QT_MAJOR_VERSION}::Qml
-                      ${LIBXAU_LINK_LIBRARIES})
+                      ${LIBXAU_LINK_LIBRARIES}
+                      ${JOURNALD_LIBRARIES})
```

Comment:
Unconditional?

Reply:
As far as I know, by not doing the `find_package` those variables should be empty. Hence, including them unconditionally would have no effect.